include/deal.II/matrix_free/fe_evaluation.h

```diff
@@ -2763,6 +2764,14 @@ class FEEvaluation : public FEEvaluationAccess<dim,
   void
   reinit(const typename Triangulation<dim>::cell_iterator &cell);
 
+  /**
+   * Prints a summary of this class (e.g., degrees with fast code path)
+   * to the given output stream.
+   */
+  template <typename StreamType>
+  void
+  print(StreamType &out) const;
```

Comment:
Maybe call the function `print_summary()` if that is what it does? I would expect a function called `print()` to output the entire *state* of the object.

Reply:
Separately, I would like it much better if the function simply took `std::ostream` as argument.